app/Http/Controllers/Wiki/LanguageControllerV2.php

```diff
@@ -168,7 +168,7 @@ public function countryLang()
             $cache_params,
             now()->addDay(),
             function () use ($sort_by, $lang_code, $country_code, $additional, $img_size, $img_type, $key) {
-                $language_v2 = $this->getV2Language($lang_code);
+                $language_v2 = !empty($lang_code) ? $this->getV2Language($lang_code) : null;
```

Comment:
If the $lang_code is null we should avoid to pull the first language record by default.